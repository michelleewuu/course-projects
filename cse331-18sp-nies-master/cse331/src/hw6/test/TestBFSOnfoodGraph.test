# Test BFS on food graph

# build the graph
LoadGraph food food.tsv

# test BFS order

# find the shortest path FruitTart to CarrotCake
FindPath food FruitTart CarrotCake

# find the shortest path CarrotCake to FruitTart
FindPath food CarrotCake FruitTart