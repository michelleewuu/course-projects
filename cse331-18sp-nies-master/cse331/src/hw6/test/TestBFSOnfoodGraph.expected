# Test BFS on food graph

# build the graph
loaded graph food

# test BFS order

# find the shortest path FruitTart to CarrotCake
path from FruitTart to CarrotCake:
FruitTart to CheeseCake via Not Vegan Dessert
CheeseCake to CarrotCake via American Dessert

# find the shortest path CarrotCake to FruitTart
path from CarrotCake to FruitTart:
CarrotCake to CheeseCake via American Dessert
CheeseCake to FruitTart via Not Vegan Dessert